Gems/MultiplayerCompression/Code/Source/LZ4Compressor.cpp

```diff
@@ -56,11 +56,16 @@ namespace MultiplayerCompression
         AZ_Warning("Multiplayer Compressor", compDataSize >= compWorstCaseSize, "Outbuffer size (%lu B) passed to Compress() is less than estimated worst case (%lu B)", compDataSize, compWorstCaseSize);
 
         // Note that this returns a non-negative int so we are narrowing into a size_t here
-        compSize = LZ4_compressHC(reinterpret_cast<const char*>(uncompData), reinterpret_cast<char*>(compData), static_cast<int>(uncompSize));
+        compSize = LZ4_compress_HC(
+            reinterpret_cast<const char*>(uncompData), 
+            reinterpret_cast<char*>(compData), 
+            static_cast<int>(uncompSize),
+            static_cast<int>(compDataSize),
+            0);
```

Comment:
0 seems good here, I took a look through the LZ4 API and it will effectively swap this to a "middle of the road" level.

Reply:
I took a look at what the old LZ4_compressHC does, and it uses the new function with 0 for this value. So we shouldn't see a behavior change.